packages/server/src/api/controllers/row.js

```diff
@@ -67,12 +67,18 @@ exports.patch = async function(ctx) {
   ctx.message = `${table.name} updated successfully.`
 }
 
-exports.save = async function(ctx) {
+exports.save = async function (ctx) {
   const instanceId = ctx.user.instanceId
   const db = new CouchDB(instanceId)
   let row = ctx.request.body
   row.tableId = ctx.params.tableId
 
+  if (ctx.request.body.type === "delete") {
```

Comment:
This code was actually here before but was lost in the merge - the new grid component relies on it.

I think the reason it was done like this is because DELETE doesn't allow a request body, so rather than pass a potentially huge array of ids as a query params the POST makes more sense.